CONTRIBUTING.md

```diff
@@ -94,14 +94,36 @@ open http://localhost:8000/
 
 To publish the packages to npm, you'll need to be added as an owner for the
 packages you're publishing. Use the #design-system Slack channel for more
-information.
+information. 
+
+Before Publishing:
+
+- Know that publishing is very easy once you have access to do so **Please Be Careful** 🤗
+- As of v2 you must be on Node v8+
+- name sure you are logged into NPM properly, if you have multiple logins you can use something like [npmrc]
 
 These Lerna commands can be helpful when publishing:
 
 - Run `npm run changed` to see which packages have changed since the last
   release.
 - Run `npm run diff` to see a diff of all packages since the last release.
 - Run `npm run publish` to publish the updated packages with Lerna.
```

Comment:
Would you mind removing the periods of these bullets, before the last bullet? (think it'd be consistent with the format of other bullet list.)